runtime/parachains/src/configuration/migration.rs

```diff
@@ -0,0 +1,284 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! A module that is responsible for migration of storage.
+
+use crate::configuration::{self, Config, Pallet, Store};
+use frame_support::{pallet_prelude::*, traits::StorageVersion, weights::Weight};
+use frame_system::pallet_prelude::BlockNumberFor;
+
+/// The current storage version.
+pub const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
+
+/// Call this during an next runtime upgrade for this module.
+pub fn on_runtime_upgrade<T: Config>() -> Weight {
+	let mut weight = 0;
+
+	if StorageVersion::get::<Pallet<T>>() == 0 {
+		weight += migrate_to_v1::<T>();
+		StorageVersion::new(1).put::<Pallet<T>>();
```

Comment:
I think it would be better to leave it like this. The reason is: here we want to migrate from version 0 to version 1, not to the current storage version. The difference is subtle, but it boils down that this is more robust to changes.

If the constant was bumped, there is a chance that it won't be noticed to be updated here. The fix would be to replace `STORAGE_VERSION` with `StorageVersion::new(1)` which is what we see here already. 

And if you look from the opposite side, we do not want this number to be updated because it will always be a migration from version 0 to version 1.